### PR TITLE
test

### PR DESCRIPTION
**Pull request by LesleyLai**

### CODE REVIEW

**Comment by FixrbotMock** [comment]
Fixrbot: [2 error]
１. [app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java] Incomplete pattern inside `userList` method
2. ...

File: app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java [hunk 1]
```diff
@@ -106,8 +106,8 @@ public boolean insert(UserStructure us){
         } catch (Exception e) {
```
Comment (FixrbotMock) [inline app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:106]: Potential missing `cursor.close()` method

```suggestion
        cursor.close();
        } catch (Exception e) {
```

**Review by FixrbotMock — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java (hunk 1 above)
Comment (LesleyLai) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:106]: fixrbot pattern

Comment (FixrbotMock) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:106]: > fixrbot pattern

API Methods called in the pattern:

```
Cursor.MoveToFirst()
Cursor.getString()
Cursor.close()
```
Pattern popularity: 94
Edit Distance: 3

7 examples on the above pattern.

Comment (LesleyLai) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:106]: fixrbot example

Comment (FixrbotMock) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:106]: > fixrbot example

- **asantibanez/Patio/**[src/main/java/com/andressantibanez/android/patio/PatioUtils.java:getRealPathFromURI](https://github.com/asantibanez/Patio/blob/2596a912c7302db7e86a8ae13aeb3e8958ce8532/Patio/src/main/java/com/andressantibanez/android/patio/PatioUtils.java#L17-L27)
- Another example

**Pull request merged**